core-common/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -34,6 +34,7 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.CLASS)
 public @interface Mapper {
+    String DEFAULT_IMPL_NAME = "*Impl";
```

Comment:
I like the idea of the wild card mechanism, but I find the "*" a bit unintuitive. I'd prefer to use a more descriptive placeholder, such "<MAPPER_NAME>".